website/docs/getting-started/using-podman-mac-helper-to-migrate-from-docker-to-podman-on-macos.md

```diff
@@ -0,0 +1,68 @@
+---
+id: using-podman-mac-helper-to-migrate-from-docker-to-podman-on-macos.md
+title: Using the `podman-mac-helper` tool on macOS
+description: Using the `podman-mac-helper` tool can make it easier to migrate from Docker to Podman on macOS, as it allows you to continue using familiar Docker commands while taking advantage of the benefits of Podman.
+tags: [podman-desktop, getting-started, podman-mac-helper, macOS]
+keywords: [podman desktop, podman, containers]
+---
+
+## Using the `podman-mac-helper` tool to migrate from Docker to Podman on macOS
```

Comment:
I think in the intro it would be nice to start with the intro on why somebody may want to do this. (which you have bellow). I'd also add emphasis that, it enables tools which interact with Docker, to communicate with Podman instead. (Maven, test containers...)

Reply:
Better?

Reply:
yes 👍